x-pack/test/functional/services/transform/wizard.ts

```diff
@@ -241,6 +242,15 @@ export function TransformWizardProvider({ getService, getPageObjects }: FtrProvi
     },
 
     async assertIndexPreviewHistogramCharts(expectedHistogramCharts: HistogramCharts) {
+      if (process.env.TEST_CLOUD) {
+        log.warning('Not running color assertions in cloud');
+        return;
+      }
+      if (process.platform === 'win32') {
```

Comment:
As above, the assertions on the histograms are now passing on Windows on master, so this isn't needed after all.

Reply:
Windows check removed in 7a5b8af1cb9